src/pair.ts

```diff
@@ -17,21 +17,23 @@ export abstract class Pair {
 	public pairKey: string | null = null;
 	public tokenA: Address = "";
 	public tokenB: Address = "";
-	protected swappaPairAddress: Address = "";
+	private swappaPairAddress: Address = "";
```

Comment:
make private now that the child classes do not need to set this directly anymore